src/Avalonia.Base/Reactive/Operators/Switch.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+
+namespace Avalonia.Reactive.Operators;
+
+// Code based on https://github.com/dotnet/reactive/blob/main/Rx.NET/Source/src/System.Reactive/Linq/Observable/Switch.cs
```

Comment:
It would be nice to add such comments to other imported sources

Reply:
There is not too much code copied with little changes. Switch, CombineLatest and Sink classes mostly.
Other used operations are way simplified for the Avalonia. For instance, Take/Skip operators are simplified and do not support combination of multiple Take/Skip calls in a row. Also, no TimeSpan support as an argument.

Reply:
Added same link comment to the Sink file. And added System.Reactive to the Notice.md.